README.md

```diff
@@ -1,111 +1,15 @@
-# ComprehendCustomerScripts
+# Amazon Comprehend Examples
 
 ## Introduction
```

Comment:
I had merged Mona's pull request yesterday which had changes in the Readme. Did you pull in those changes before raising this PR?